Exceptions/CloudApiQuotaExceeded.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Piwik\Plugins\GoogleAnalyticsImporter\Exceptions;
+
+class CloudApiQuotaExceeded extends \RuntimeException
+{
+    public function __construct($limit = 0)
+    {
+        parent::__construct('Daily cloud rate limit reached, try again tomorrow. (Note: GA by default sets a daily limit on the number'
+            . ' of API requests made each day. This has been prorated to  ' . $limit . 'requests to your account. 
```

Comment:
@elabuwa There is a space missing before `requests`

Reply:
fixed